src/components/shared/references/StudiesTable.tsx

```diff
@@ -2,13 +2,13 @@ import _ from "lodash";
 import React, { useEffect, useState } from "react";
 import { useMediaQuery } from "react-responsive";
 import { Dropdown, DropdownProps, Pagination, Table } from "semantic-ui-react";
-import { mobileDeviceWidth } from "../../constants";
-import { AirtableRecord } from "../../types";
-import Translate from "../../utils/translate/translateService";
-import { getGeography, getPossibleNullDateString } from "../../utils/utils";
-import StudyDetailsModal from "../charts/StudyDetailsModal";
-import ReferenceSearch from '../charts/ReferencesSearch';
-import { sendAnalyticsEvent } from '../../utils/analyticsUtils';
+import { mobileDeviceWidth } from "../../../constants";
+import { AirtableRecord } from "../../../types";
+import Translate from "../../../utils/translate/translateService";
+import { getGeography, getPossibleNullDateString } from "../../../utils/utils";
+import StudyDetailsModal from "./StudyDetailsModal";
+import ReferenceSearch from './ReferencesSearch';
+import { sendAnalyticsEvent } from '../../../utils/analyticsUtils';
```

Comment:
Not totally necessary, but we could make imports absolute paths. It makes it cleaner, especially if they are in the source directory ie: `import { mobileDeviceWidth } from "constants";`